participants/iobert.json

```
@@ -0,0 +1,24 @@
+{
+	"realName": {
+		"givenName": "Marius",
+		"familyName": "Obert",
+		"placeFamilyNameFirst": false,
+		"hideFamilyNameOnWebsite": false
+	},
+	"githubAccountName": "iobert",
+	"company": "Twilio",
+	"when": {
+		"friday": true,
+		"saturday": false
+	},
+	"iCanTakeNotesDuringSessions": false,
+	"tags": ["React", "NextJS", "AI", "", "Communication APIs"],
```

Not sure whether an empty tag is working... 🤔

```suggestion
	"tags": ["React", "NextJS", "AI", "Communication APIs"],
```